en/docs/deploy/elk-analytics-sso-guide.md

```diff
@@ -2,8 +2,11 @@
  
  You can enable Single-sign-on for an ElasticSearch/Kibana deployment with WSO2 Identity Server via OpenID Connect. To set this up, follow the steps given below.
  
-!!! note "Prerequisite"
-          An Elasticsearch platinum subscription is required to enable SSO in ELK.
+!!! note "Prerequisites"
```

Comment:
Why not have prerequisites as a separate heading?

Reply:
It's just one prerequisite. Do you think it's better?